README.md

```diff
@@ -8,44 +8,87 @@ interactive dashboard:
 
 ![Alt Text](/docs/cord19_semantic_clusters.gif)
 
+
+## Getting started
+
+The packaged is called `semantic_clustering` and can be installed from the download folder with:
+
+```shell
+pip install .
+```
+or from the command line directly with:
+
+```shell
+python -m pip install git+https://github.com/asreview/semantic-clusters.git
+```
+
+### Commands
+
+For help use:
+
+```shell
+asreview semantic_clustering -h
+asreview semantic_clustering --help
+```
+
+Other options are:
+
+```shell
+asreview semantic_clustering -f <input.csv or url> -o <output.csv>
+asreview semantic_clustering --filepath <input.csv or url> --output <output.csv>
+```
+
+```shell
+asreview semantic_clustering -t -o <output.csv>
+asreview semantic_clustering --testfile --output <output.csv>
+```
+
+```shell
+asreview semantic_clustering -a <output.csv>
+asreview semantic_clustering --app <output.csv>
+```
+
+```shell
+asreview semantic_clustering -v
+asreview semantic_clustering --version
+```
+
+
 ## Usage
-The usage of the semantic clustering app is found in the main.py file. The
+The functionality of the semantic clustering extension is implemented in a [subcommand extension](https://asreview.readthedocs.io/en/latest/API/extension_dev.html#subcommand-extensions). The
 following commands can be run:
 
 ### Processing
-```console
-python asreviewcontrib\semantic_clustering\main.py -f <url or local file>
-python asreviewcontrib\semantic_clustering\main.py --filepath <url or local file>
+In the processing phase, a dataset is processed and clustered for use in the interactive interface. The following options are available:
+
+```shell
+asreview semantic_clustering -f <input.csv or url> -o <output_file.csv>
+asreview semantic_clustering -t -o <output_file.csv>
 ```
 
-The filepath argument starts the processing of a file for clustering. This file
-will be saved to the `data` folder after the processing is done. An example of
-usage can be:
+Using `-f` will process a file and store the results in the file specified in `-o`. 
 
-```console
-python asreviewcontrib\semantic_clustering\main.py -f "https://raw.githubusercontent.com/asreview/systematic-review-datasets/master/datasets/van_de_Schoot_2017/output/van_de_Schoot_2017.csv"
-```
+Using `-t` instead of `-f` uses the [`van_de_Schoot_2017`](https://asreview.readthedocs.io/en/latest/intro/datasets.html?highlight=ptsd#featured-datasets) dataset instead. 
```

Comment:
```suggestion
Using `-t` instead of `-f` uses the [`van_de_Schoot_2017`](https://asreview.readthedocs.io/en/latest/intro/datasets.html#featured-datasets) dataset instead. 
```

Reply:
Is there a reason for this? It should link to the ptsd dataset right?